boards/arm/nrf5340dk_nrf5340/nrf5340_cpuapp_common.dts

```diff
@@ -101,6 +101,11 @@
 				 <5 &adc 5>;	/* A5 = P0.26 = AIN5 */
 	};
 
+	gpio_fwd: nrf-gpio-forwarder {
+		compatible = "nordic,nrf-gpio-forwarder";
+		status = "okay";
+	};
```

Comment:
What is the point in defining and enabling the node without specifying any pins to be forwarded (BTW currently, the commit message says "for nRF5340, leave the node with status disabled by default", what is in fact misleading)? I think that either the following pins should be specified here: https://github.com/zephyrproject-rtos/zephyr/blob/d353e551607926612ff258dadbc13d028d58b54e/boards/arm/nrf5340dk_nrf5340/nrf5340_cpunet_reset.c#L16-L20
(and `nrf5340_cpunet_reset.c` updated accordingly) or all changes related to the nRF5340 DK board should be left for a follow-up PR.

Reply:
At first, the node was in `"disabled"` state by default. However, after this comment: https://github.com/zephyrproject-rtos/zephyr/pull/42576#discussion_r802777879 and the discussion that followed, I decided to change the default status to `"okay"`.

I think that the best way to handle the issue is to have the pins you mentioned added to the board's DTS file and leave default status `"okay"`. What do you think?

Reply:
> I think that the best way to handle the issue is to have the pins you mentioned added to the board's DTS file and leave default status `"okay"`. What do you think?

If you are willing to do it in this PR, sure, that works for me.

Reply:
Done, please have a look if it's alright now

Reply:
Looks good. Thank you!